pallets/funding/src/mock.rs

```diff
@@ -398,9 +398,9 @@ impl Config for TestRuntime {
 	type InvestorOrigin = EnsureInvestor<TestRuntime>;
 	type ManualAcceptanceDuration = ManualAcceptanceDuration;
 	type MaxBidsPerProject = ConstU32<1024>;
-	type MaxBidsPerUser = ConstU32<4>;
+	type MaxBidsPerUser = ConstU32<25>;
```

Comment:
I discussed with Luca that we will limit the MaxBidsPerUser and MaxContributionsPerUser to 16.

Reply:
Ok, but the mock needs more for the multiplier tests